frontend/src/main.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
-import store from './store'
+import store from './store/index'
```

Comment:
you don't need to specify `index` here. it automatically looks for an index file in the `store` folder